sr_ethercat_hand_config/controls/host/lh/sr_edc_effort_controllers.yaml

```diff
@@ -1,118 +1,118 @@
 # First Finger (Index finger)
 sh_lh_ffj0_effort_controller:
   type: sr_mechanism_controllers/SrhEffortJointController
-  joint:  lh_FFJ0
-  max_force: 0
+  joint: lh_FFJ0
+  max_force: 600
```

Comment:
I assume this will be safe, as this controller does not depend on the hand being calibrated (it just sends PWM).

Reply:
Yes, you are right this control is not using any calibration.